book/invalidation.md

```diff
@@ -5,7 +5,7 @@ prev: embeds
 next: skipped
 ...
 
-Compositing makes animations smoother, but
+Compositing (See Chapter 13) makes animations smoother, but
```

Comment:
Do we really want this? also shouldn't "See" be lower case?

Reply:
appears to work